index.html

```diff
@@ -34,7 +32,142 @@ <h2 class="title">I'm Mariane Glad to see you!</h2>
+            <li class="back">2015</li>
+        </ul>
+
+        <p class="more-info">A daily selection of privately 
+            personalized reads; no accounts or sign-ups required.
+        </p>
+
+        <ul class="languages">
+            <li><button class="lang" id="lan-1">html</button></li>
+            <li><button class="lang" id="lan-2">css</button></li>
+            <li><button class="lang" id="lan-3">javaScript</li></button></li>
+        </ul>
 
+        <button class="see" id="see-1">See Project</button>
     </section>
+
+    <section class="works-1">
+        <img src="images/Snapshoot Portfolio (2).svg" alt="" class="tonic">
+        <h1 class="stories">Multi-Post Stories</h1>
+
+        <ul class="description">
+            <li class="canopy">CANOPY</li>
+            <li><img src="images/Counter.svg" alt="" id="counter"></li>
+            <li class="back">Back End Dev</li>
+            <li><img src="images/Counter.svg" alt="" id="counter"></li>
+            <li class="back">2015</li>
+        </ul>
+
+        <p class="more-info">A daily selection of privately 
+            personalized reads; no accounts or sign-ups required.
+        </p>
+
+        <ul class="languages">
+            <li><button class="lang" id="lan-1">html</button></li>
+            <li><button class="lang" id="lan-2">css</button></li>
+            <li><button class="lang" id="lan-3">javaScript</li></button></li>
+        </ul>
+
+        <button class="see" id="see-1">See Project</button>
+    </section>
+
+    <section class="myself">
+        <h1 class="about">About Myself</h1>
+        <p class="experience">Hello I'm a software developer! 
+            I can help you build a product , feature or website Look 
+            through some of my work and experience! If you like what you 
+            see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+
+        <p class="connect-1">LET'S CONNECT</p>
+
+        <ul class="icons-1">
+            <li><img src="images/Vector.png" class="icons-2"></li>
+            <li><img src="images/Vector (1).png" class="icons-2"></li>
+            <li><img src="images/Vector (2).png" class="icons"></li>
+            <li><img src="images/Vector (3).png" class="icons-2"></li>
+            <li><img src="images/Vector (4).png" class="icons-2"></li>
+            <li><img src="images/Vector (5).png" class="icons-2"></li>
+        </ul>
+
+        <button class="resume" id="resume-1">Get my resume</button>
+
+        <h2 class="languages-1">Languages</h2>
+
+        <ul class="language-icons">
+            <li><button class="button-1"><img src="images/javaScript.svg" class="js">Javascript</button></li>
+            <li><button class="button-2"><img src="images/HTML.svg" class="html">HTML</button></li>
+            <li><button class="button-3"><img src="images/CSS3.svg" class="css">CSS</button></li>
+        </ul>
+
+        <p class="frame">Frameworks</p>
+        <p class="skills">Skills</p>
+
```

Comment:
- [ ] This part of your  `about myself` section doesn't look like the Figma design. It is missing the arrow icons for the `languages`, `skills`, and `frameworks` and the current styles are a bit different from the Figma design. 
- [ ] I will suggest that you include the icons and style this part properly to meet the Figma design guidelines :100:  
- Your Design
![image](https://user-images.githubusercontent.com/102699958/219921810-7f2b4308-8e24-464f-a440-92368bea503c.png)

- The Figma design
![image](https://user-images.githubusercontent.com/102699958/219921866-1613e2bb-3fa8-4963-a5d9-10609267fb45.png)